selinux/bluechi.fc

```diff
@@ -1,4 +1,4 @@
-/usr/bin/bluechi-controller	--	gen_context(system_u:object_r:bluechi_exec_t,s0)
+/usr/libexec/bluechi-controller	--	gen_context(system_u:object_r:bluechi_exec_t,s0)
```

Comment:
Please leave both paths.  Certain Distros do not support libexec.

Reply:
So both path will be available for selinux configuration, but from spec point of view we will use only libexec?

Reply:
@mwperina seems correct.